### ISSUE TITLE
[FEATURE] Tactical animations

### ISSUE THREAD

**Issue by SfortzaPhantom** [Paneedah/paneedah-modern-warfare#61]
**Is your feature request related to a problem? Please describe.**
So there is a mod called Modular Movements, which adds several types of movement, like leaning or proning. This mod is heavily incompatible with Modern Warfare because of Modern Warfare's code principles. A big issue, if you'd ask me.

**Describe the solution you'd like**
I'd love to have leaning, sitting and proning animations in Modern Warfare. Or, at least, compatibility with Modular Movements.

**Describe alternatives you've considered**
Using Modular Warfare instead of Modern Warfare.

**Additional context**
-

**Comment by DreamSky7** [comment]
What is the incompatibility of modular movement？

**Comment by Oddlaceguy49** [comment]
When you try to lean to the left or put the gun in your offhand, the textures get very messed up

**Comment by SfortzaPhantom** [comment]
To be fair, there is an another incompatibility, VMW guns can't shoot properly when you use Modular Movements.

How it works with MW? You lean to the left and you can shoot from the corner. Bullets will sync with your position. If you'll try to do the same with VMW, it'll lead to shooting the wall.

I'd love to see sort of compatibility between these two mods - in positioning with 3D armor from VMW and so-on - but I assume it'll be a tough job and require to rewrite the code of VMW completely.

**Comment by Oddlaceguy49** [comment]
Yeah I noticed that but forgot to mention it. It would be very cool though to at least see the slide move come into the game.

**Comment by DreamSky7** [comment]
The more reason is the modular action itself, not VMW

**Comment by DreamSky7** [comment]
Of course, the bullet is shot from the center of the screen. This is the problem of VMW

**Comment by Desoroxxx** [comment]
Hey, we recently moved repository; would you be so kind to create this feature request [here]?

[here]: https://github.com/Paneedah/Modern-Warfare-Cubed/issues
